docs/0install/setup.md

```diff
@@ -0,0 +1,28 @@
+The 0install distribution of the client is to allow zanata-cli to be available on multiple platform. This distribution requires [0install](http://0install.net/) to run.
+
+## Prerequisites
+
+1. Install [0install](http://0install.net/injector.html) on your machine.
```

Comment:
Right, but the disadvantage is that we are inconveniencing all users
because of a Mac problem, when the aim of this work is to make things
simpler for users.

Anyway, didn't Luke succeed in using the feed even when it depended on
http://repo.roscidus.com/java/openjdk-jre, by installing the JDK manually?

On 22 June 2015 at 12:24, Alex Eng <notifications@github.com> wrote:

> In docs/0install/setup.md
> <https://github.com/zanata/zanata-client/pull/68#discussion_r32902028>:
>
> > @@ -0,0 +1,28 @@
> > +The 0install distribution of the client is to allow zanata-cli to be available on multiple platform. This distribution requires [0install](http://0install.net/) to run.
> > +
> > +## Prerequisites
> > +
> > +1. Install [0install](http://0install.net/injector.html) on your machine.
>
> The advantage of having single feeds file is when the dependency is fix,
> mac user just need to do 0install update zanata-cli. If separated, we will
> need to force them to change url again.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-client/pull/68/files#r32902028>.
>